mount/mode/mode.go

```diff
@@ -0,0 +1,80 @@
+package mode
```

Comment:
You can't do this, because we need access to the Mount.Mode within the label.go which is included by mount, so we end up with a circular dependency.